jsonfield/fields.py

```diff
@@ -78,17 +78,17 @@ def db_type(self, connection):
         if connection.vendor == 'mysql':
             return 'longtext'
         if connection.vendor == 'oracle':
-            return 'long'
+            return 'clob'
```

Comment:
can we use an extra kwargs in `__init__` instead ?

like

```python
def __init__(self, *args, **kwargs):
  self.custom_db_type = kwargs.pop('db_type', None)

def db_type(self, connection):
  if self.custom_db_type:
    return self.custom_db_type

Reply:
Yes, but I'd change the default type for Oracle anyway. `long` is deprecated and have big limitations: for example, you cannot have more than one `long` column in a table.

Reply:
@mpaolini Added the option to customize `db_type` as you suggested.